base/constants.go

```diff
@@ -59,8 +59,6 @@ const (
 
 	DefaultDropIndexes = false // Whether Sync Gateway drops GSI indexes before each test while running in integration mode
 
-	DefaultOldRevExpirySeconds = uint32(300)
```

Comment:
As discussed, leave this intact for now to support mix of delta and non-delta clients, even if SG is running in delta mode.